docs/about/overview/encryption.mdx

```diff
@@ -7,103 +7,112 @@ sidebar_position: 3
-- Accidently sharing the channel key with the wrong person.
+This means Meshtastic is vulnerable to [**« Harvest now, Decrypt later »**](https://en.wikipedia.org/wiki/Harvest_now,_decrypt_later) attacks. **This attack itself is not enough to decrypt messages**; it relies on another failure leaking the channel key, such as:
+- Accidentally sharing the channel key with the wrong person.
 - One of your nodes being stolen.
 - Any unknown bug revealing the channel key to an attacker.
 
-Other protocols such as *misconfigured TLS1.2* are far more impacted by Harvest now, Decrypt later attacks because their keys will be broken by near future Quantum-Computers yet they do not provide PFS.
+Other protocols, such as *misconfigured TLS1.2*, are far more impacted by Harvest now, Decrypt later attacks because their keys will be broken by near-future Quantum-Computers, yet they do not provide PFS.
 
-Meshtastic's encryption is not threatened by Quantum-Computers\* so realistic attack vectors are accidental miss-handling of channel keys and nodes being lost.
+Meshtastic's encryption is not threatened by Quantum-Computers\*, so realistic attack vectors are accidental mishandling of channel keys and nodes being lost.
 
-\*on the Quantum-Resistance of AES256 see:
-- [this stackoverflow question, contain a bit of debates and two good answers](https://crypto.stackexchange.com/q/6712)
+\*On the Quantum-Resistance of AES256, see:
+- [this stackoverflow question, containing a bit of debate and two good answers](https://crypto.stackexchange.com/q/6712)
 - [NIST's Post-Quantum FAQ](https://csrc.nist.gov/projects/post-quantum-cryptography/faqs) section « To protect against the threat of quantum computers, should we double the key length for AES now? (added 11/18/18) »
 
 **Recommendations** for users using *private channels*:
-- **Do not configure private channels on unattended nodes**, nodes will relay meshtastic traffic even if they are not able to decrypt it.
-  Your unattended routers should not have the private PSK configured as it is easy to gain physical access and extract the channel key.
-- Keep in mind, everything sent on a channel can be stored and decrypted later by anyone who gain access to the key even if you locally delete the messages.
+- **Do not configure private channels on unattended nodes**. Nodes will relay Meshtastic traffic even if they are not able to decrypt it. Your unattended routers should not have the private PSK configured as it is easy to gain physical access and extract the channel key.
+- Keep in mind that everything sent on a channel can be stored and decrypted later by anyone who gains access to the key, even if you locally delete the messages.
 - Change your Channel Keys from time to time.
 
 ### Integrity
 
 Integrity means ensuring messages cannot be modified without the key.
```

Comment:
This section needs qualifiers added.
With 2.5:
- DMs are now protected
- 2.5 PKI Admin messages are now protected.

(I think, maybe, *probably*, I did**n't** checked please do.)